diff --git a/config/win/BUILD.gn b/config/win/BUILD.gn
index de1c0f190..069432bba 100644
--- a/config/win/BUILD.gn
+++ b/config/win/BUILD.gn
@@ -291,7 +291,7 @@ config("runtime_library") {
 # manually override this config for their compiles.
 config("winver") {
   defines = [
-    "NTDDI_VERSION=NTDDI_WIN11_BR",
+    "NTDDI_VERSION=NTDDI_WIN11_GE",
 
     # We can't say `=_WIN32_WINNT_WIN10` here because some files do
     # `#if WINVER < 0x0600` without including windows.h before,
